src/devices/sound/aica.cpp

```diff
@@ -8,12 +8,14 @@
     - No FM mode
     - A third sample format (ADPCM) has been added
     - Some minor other tweeks (no EGHOLD, slighly more capable DSP)
+	- Internal ARM7DI Core
 */
 
 #include "emu.h"
 #include "aica.h"
 
 #define ICLIP16(x) (x<-32768)?-32768:((x>32767)?32767:x)
+#define CLIP(x,min,max) (x<min)?min:((x>max)?max:x)
```

Comment:
This isn't safe with macro substitution when the arguments are expressions.  The macro above it isn't safe either, but there's no excuse for adding more unsafe macros.